frontend/src/components/CostTable.vue

```diff
@@ -182,11 +188,32 @@ watch(
   }
 );
 
+watch(
+  () => props.minVCPU,
+  () => {
+    refreshDataTable();
+  }
+);
+
+watch(
+  () => props.minRAM,
+  () => {
+    refreshDataTable();
+  }
+);
+
 const refreshDataTable = () => {
   state.dataRow = [];
   let rowCnt = 0;
   const selectedRegionSet = new Set<string>([...props.regionList]);
   props.dbInstanceList.forEach((dbInstance) => {
+    if (
+      Number(dbInstance.memory) < props.minRAM ||
```

Comment:
qq: any reason dbInstance.memory wasn't Number?

Reply:
Thia may be the consequence of contributing without code review😢. 
I will fix this by another PR.